src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -130,6 +141,34 @@ def __init__(self, msConfig, logger=None):
         self.wfCounters = {'cleaned': {},
                            'archived': {'normalArchived': 0,
                                         'forceArchived': 0}}
+        self.globalLocks = set()
+
+    def getGlobalLocks(self):
+        """
+        Fetches the list of 'globalLocks' from wmstats server and the list of
+        'parentLocks' from request manager. Stores/updates the unified set in
+        the 'globalLocks' instance variable. Returns the resultant unified set.
+        :return: A union set of the 'globalLocks' and the 'parentLocks' lists
+        """
+        globalLocks = set()
+        parentLocks = set()
+        self.logger.info("Fetching globalLocks list from wmstats server.")
+        try:
+            globalLocks.update(set(self.wmstatsSvc.getGlobalLocks()))
+        except Exception as ex:
+            msg = "Failed to refresh global locks list for the current polling cycle. Error: %s\n "
```

Comment:
Can you please keep the error message in the same line? So, removing the line breaker `\n` from it.

Reply:
Done